superset/config.py

```diff
@@ -522,7 +522,7 @@ class CeleryConfig(object):
 SMTP_MAIL_FROM = "superset@superset.com"
 
 if not CACHE_DEFAULT_TIMEOUT:
-    CACHE_DEFAULT_TIMEOUT = CACHE_CONFIG.get("CACHE_DEFAULT_TIMEOUT")  # type: ignore
+    CACHE_DEFAULT_TIMEOUT = CACHE_CONFIG["CACHE_DEFAULT_TIMEOUT"]  # type: ignore
```

Comment:
The parameters you mentioned in the description that needed to be defaulted (`ENABLE_CHUNK_ENCODING` etc) seem to be missing here.

Reply:
Do we still need the `# type: ignore`?